R/RcppExports.R

```diff
@@ -227,6 +227,38 @@ has_positions <- function(dbpath) {
     .Call('RXapian_has_positions', PACKAGE = 'RXapian', dbpath)
 }
 
+#' xapian_delete wrapper
+#' 
+#' @param dbpath path to a Xapian Database
+#' @param docid the document ID of the document to be removed
+#' @examples
+#' \dontrun{
+#' dbpath<- c("path/to/database")
+#' docid <- 20
+#' deleteWrapper(dbpath,docid)
+#' }
+#' 
+#' @return none
+deleteWrapper <- function(dbpath, docid) {
+    invisible(.Call('RXapian_deleteWrapper', PACKAGE = 'RXapian', dbpath, docid))
```

Comment:
I've only just noticed, but you seem to have inconsistent indentation between different `.R` files: four spaces here, two in `R/DatabaseWrapper.R`. Worth tidying up.